priv/templates/nimble_template/assets/.eslintrc.json

```diff
@@ -6,10 +6,10 @@
   },
   "extends": [
     "eslint:recommended",
-    "plugin:prettier/recommended"
+    "prettier"
```

Comment:
> Since version 8.0.0 of eslint-config-prettier, all you need to extend is "prettier"! That includes all plugins.

https://github.com/prettier/eslint-config-prettier